### ISSUE TITLE
Implement label smoothing

### ISSUE THREAD

**Issue by dcfidalgo** [recognai/biome-text#25]
If we want to implement *label smoothing* (a regularization method used for example in the original transformer paper), this link is helpful:
https://discuss.pytorch.org/t/cross-entropy-with-one-hot-targets/13580/2

I think it is a good method if we are not sure that the gold labels are 100% correct!

**Comment by dvsrepo** [comment]
It would be nice to include it and test it! It's a good match for feedback
generated datasets.
